scanners/starttls.py

```diff
@@ -0,0 +1,150 @@
+from datetime import datetime, timedelta
+import logging
+import re
+import requests
+import time
+
+command = None
+init = None
+
+
+def starttls_check_url(domain):
+    return 'https://starttls.info/api/check/%s' % domain
+
+
+def poll_starttls_info_status(domain):
+    """
+    Poll the starttls.info API endpoint until the status of the info for a
+    given `domain` is 'DONE'.
+
+    We poll every 5 seconds.
+
+    # TODO: set a maximum poll time after which we stop trying and report an error
```

Comment:
Even if I merge this, this `TODO` about setting up a max poll time is a dealbreaker for me using the scanner -- I get very nervous about `while True` lines.